doc/manual/decoding.rst

```diff
@@ -27,6 +27,10 @@ Scikit-learn API enforces the requirement that data arrays must be 2D. A common
 
 To recover the original 3D data, an ``inverse_transform`` can be used. The ``epochs_vectorizer`` is particularly useful when constructing a pipeline object (used mainly for parameter search and cross validation). The ``epochs_vectorizer`` is the first estimator in the pipeline enabling estimators downstream to be more advanced estimators implemented in Scikit-learn. 
 
+Vectorizer
+^^^^^^^^^^
+Scikit-learn API provides functionality to chain transformers and estimators by using :class:`sklearn.pipeline.Pipeline`. We can construct decoding pipeline and perform cross-validation and grid-search. However scikit-Learn transformers and estimators generally expect 2D data (n_samples * n_features), whereas MNE transformers typically output data with higher dimensionality (e.g. n_samples * n_channels * n_frequencies * n_times). A Vectorizer therefore needs to be applied between the MNE and the Scikit-Learn steps: e.g: make_pipeline(Xdawn(), Vectorizer(), LogisticRegression())
```

Comment:
'We can construct a decoding pipeline' or 'We can construct decoding pipelines'

'with a higher dimensionality'. 

'scikit-learn' (not upper case)